changelog.md

```diff
@@ -1,4 +1,5 @@
 # Future
+- [FIXED] Removed `Sequelize.literal()` support for order strings [#6935](https://github.com/sequelize/sequelize/issues/6935)
```

Comment:
This is the exact opposite of what we want. `Sequelize.literal()` must still be supported, only passing a string should be removed.